src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow.tsx

```diff
@@ -271,9 +185,41 @@ export default class SentryApplicationRow extends React.PureComponent<Props> {
       return isInstalled ? this.renderUninstallButton() : this.renderInstallButton();
     }
 
-    return app.status === 'published'
-      ? this.renderPublishedAppButtons()
-      : this.renderUnpublishedAppButtons();
+    return (
+      <Access access={['org:admin']}>
```

Comment:
Thoughts on moving the entire `renderButtons` method to its own file + component?